src/accessors/external_accessor.ts

```diff
@@ -831,6 +831,26 @@ export class ExternalAccessor implements IManagementApiAccessor, IManagementSock
     return httpResponse.result;
   }
 
+  public async getProcessInstanceLog(
+    identity: IIdentity,
+    processModelId: string,
+    processInstanceId: string,
+    ): Promise<Array<DataModels.Logging.LogEntry>> {
```

Comment:
Indentation is a bit off here.

```suggestion
  ): Promise<Array<DataModels.Logging.LogEntry>> {
```